pkgs/os-specific/linux/zfs/default.nix

```diff
@@ -250,19 +256,11 @@ in {
     # IMPORTANT: Always use a tagged release candidate or commits from the
     # zfs-<version>-staging branch, because this is tested by the OpenZFS
     # maintainers.
-    version = "2.1.10-staging-2023-03-15";
-    rev = "a5c469c5f380b09705ad0bee15e2ca7a5f78213c";
+    version = "2.1.12-staging-2023-04-18";
```

Comment:
I just realized that the unstable bump was not correctly separated in another commit, but so be it, let's not do this next time